contrib/spec/openarc.spec.in

```diff
@@ -1,17 +1,42 @@
-# Copyright (c) 2010, 2011, 2016, The Trusted Domain Project.
-# All rights reserved.
+%global commit0 0.1.0
+%global gittag0 GIT-TAG
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+%global systemd (0%{?fedora} >= 18) || (0%{?rhel} >= 7)
 
 Summary: An open source library and milter for providing ARC service
 Name: openarc
-Version: @VERSION@
-Release: 1
-License: BSD
+Version: 0.1.0
+Release: 1%{?dist}
+License: BSD and Sendmail
 Group: System Environment/Daemons
+URL: https://github.com/mskucherawy/OpenARC
+
+BuildRequires: libtool
+BuildRequires: pkgconfig(openssl)
+BuildRequires: pkgconfig(libbsd)
+
+# sendmail-devel renamed for F25+
+%if 0%{?fedora} > 25
+BuildRequires: sendmail-milter-devel
+%else
+BuildRequires: sendmail-devel
+%endif
+
+Requires: lib%{name}%{?_isa} = %{version}-%{release}
 Requires: libopenarc = %{version}-%{release}
-BuildRequires: sendmail-devel, openssl-devel
-@SPECREQUIRES@
-@SPECBUILDREQUIRES@
-Source: openarc-%{version}.tar.gz
+Requires(pre): shadow-utils
+%if %systemd
+# Required for systemd
+%{?systemd_requires}
+BuildRequires: systemd
+%else
+# Required for SysV
+Requires(post): chkconfig
+Requires(preun): chkconfig, initscripts
+Requires(postun): initscripts
+%endif
+
+Source0:  https://github.com/mskucherawy/OpenARC/archive/%{commit0}.tar.gz#/%{name}-%{shortcommit0}.tar.gz
```

Comment:
I disagree with this; releases should be versioned tarballs from `make dist`, not GitHub snapshots.